en-US/Players_Guide/03_Creating_a_Character/Level_Advancement/Feats/Surgical_Combatant.md

```diff
@@ -0,0 +1,8 @@
+##### Surgical Combatant
+
+- When you take the Attack action and make a weapon attack, as well as any additional attacks granted by Extra Attack:
```

Comment:
The colon at the end here was a full stop in the Google Doc; however that makes little sense. This section may have to be reworded.